ACE/ace/os_include/sys/os_sem.h

```diff
@@ -27,6 +27,10 @@
 
 #if !defined (ACE_LACKS_SYS_SEM_H)
 #  include /**/ <sys/sem.h>
+#else
+#  ifdef ACE_HAS_LINUX_SEM_H
```

Comment:
Looks ACE_HAS_LINUX_SEM_H is not set in any config file, is this addition required?

Reply:
No, it was leftover from an attempt to remedy to the problem with `semun` and it can be removed.

Reply:
Ok, removed with #745